soda/scientific/soda/scientific/anomaly_detection/models/prophet_model.py

```diff
@@ -396,12 +396,13 @@ def detect_anomalies(self):
         self.predictions.loc[self.predictions["real_data"] > self.predictions["yhat_upper"], "is_anomaly"] = 1
         self.predictions.loc[self.predictions["real_data"] < self.predictions["yhat_lower"], "is_anomaly"] = -1
 
+        # TODO: [CLOUD-2990] re-enable once we compute uncertainty intervals correctly
         # derive anomaly probability (by normalising delta between real and predicted)
-        self._derive_anomaly_probability(self.predictions, anomaly_flag_col="is_anomaly", real_data_colname="real_data")
+        # self._derive_anomaly_probability(self.predictions, anomaly_flag_col="is_anomaly", real_data_colname="real_data")
```

Comment:
are we sure the backend doesn't use that field to determine whether something is anomalous or not?

Are we sure **we** don't use it in the feedback?

Reply:
hey @bastienboutonnet 

> are we sure the backend doesn't use that field to determine whether something is anomalous or not?

Backend works without this field, see this [dataset on dev](https://dev.sodadata.io/checks/434b239d-da81-4262-936c-973d6495f4ee/metric-result?testResultId=badd6a54-1eaf-472b-a8fc-24ba5706776f) that was created running anomaly detection using the branch of the current PR

> Are we sure we don't use it in the feedback?

 Yes! In fact I'll remove the comment about the [gnarly conversion](https://github.com/sodadata/soda-core/blob/6a361a3eb84daf19eacbb75ef5b9518b6400cda7/soda/scientific/soda/scientific/anomaly_detection/models/prophet_model.py#L416-L419) as well (I guess it refers to something that is no longer there?). As you can see in the [_criticality_threshold_calc code](https://github.com/sodadata/soda-core/blob/6a361a3eb84daf19eacbb75ef5b9518b6400cda7/soda/scientific/soda/scientific/anomaly_detection/models/prophet_model.py#L439-L450) we only use `y_upper` and `y_lower` together with the criticality threshold which [is hardcoded to 0.1](https://github.com/sodadata/soda-core/blob/6a361a3eb84daf19eacbb75ef5b9518b6400cda7/soda/scientific/soda/scientific/anomaly_detection/detector_config.yaml#L28)